specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/masterCustomIpPrefix.json

```diff
@@ -0,0 +1,414 @@
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/masterCustomIpPrefixes": {
+      "get": {
+        "tags": [
+          "MasterCustomIPPrefixes"
+        ],
+        "operationId": "MasterCustomIPPrefixes_List",
+        "description": "Gets all master custom IP prefixes in a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of MasterCustomIPPrefix resources.",
+            "schema": {
+              "$ref": "#/definitions/MasterCustomIPPrefixListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List resource group master custom IP prefixes": {
+            "$ref": "./examples/MasterCustomIpPrefixList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MasterCustomIPPrefixPropertiesFormat": {
+      "properties": {
+        "cidr": {
+          "type": "string",
+          "description": "The prefix range in CIDR notation. Should include the start address and the prefix length."
+        },
+        "originalValidationMessage": {
+          "type": "string",
+          "description": "Original message used to request creation of master custom IP Prefix."
+        },
+        "signedValidationMessage": {
+          "type": "string",
+          "description": "Signed message used to request creation of master custom IP Prefix."
+        },
+        "geography": {
+          "type": "string",
+          "description": "Geography of edge routers this CIDR can be reflected on. Related to geographies covered by the 5 International RIRs (ARIN/RIPE/APNIC/LACNIC/AFRINIC.",
+          "enum": [
+            "NorthAmerica",
+            "Europe",
+            "Asia",
+            "SouthAmerica",
+            "Africa",
+            "Global"
+          ],
+          "x-ms-enum": {
+            "name": "RIR",
+            "modelAsString": true
+          }
+        },
+        "customIpPrefixes": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ReferencedCustomIPPrefix"
```

Comment:
Please use `./network.json#/definitions/SubResource` which is an ID reference to avoid duplicating definitions and remove `ReferencedCustomIPPrefix` from the file

Reply:
fixed